pkg/web/templates/components/add_counterparty.html

```diff
@@ -0,0 +1,116 @@
+{{ define "add_cparty_modal" }}
+
+{{ $sunrise := .SunriseEnabled }}
+
+<dialog id="add_cparty_modal" class="modal">
+  <div class="modal-box max-w-3xl max-h-screen">
+    <div class="flex justify-between items-center">
+      <h1 class="font-bold text-xl">Add New Counterparty VASP</h1>
+      <button onclick="add_cparty_modal.close()" class="btn btn-sm btn-circle btn-ghost">
+        <i class="fa-solid fa-x"></i>
+        <span class="sr-only">Close modal</span>
+      </button>
+    </div>
+    <p class="py-4">Add new counterparty VASP details.</p>
+    <div>
+      <form id="new-cparty-form" hx-post="/v1/counterparties" hx-ext="json-enc" hx-target="#counterparties" hx-swap-oob="outerHTML:#counterparties" method="post">
+        <div class="mb-4 grid gap-6 md:grid-cols-2">
+          <div>
+            <label for="name" class="label-style">VASP Name</label>
+            <input type="text" id="name" name="name" class="input-style" />
+          </div>
+          <div>
+            <label for="website" class="label-style">Website</label>
+            <input type="text" id="website" name="website" class="input-style" />
+          </div>
+        </div>
+        <div class="mb-4 grid gap-6 md:grid-cols-2">
+          <div>
+            <label for="common_name" class="label-style">Common Name</label>
+            <input type="text" id="common_name" name="common_name" class="input-style" />
+          </div>
+          <div>
+            <label for="endpoint" class="label-style">Endpoint</label>
+            <input type="text" id="endpoint" name="endpoint" class="input-style" />
+          </div>
+        </div>
+
+        {{ if $sunrise }}
```

Comment:
Would it be better to amend the modal so that we only show the compliance officer name and email inputs if the selected protocol is `Email`? We'd have to move the protocol option to the top of the modal.

Reply:
It might be - we can discuss in the Envoy UI sync.